server/athenian/api/internal/features/entries.py

```diff
@@ -1850,6 +1863,10 @@ def __sentry_repr__(self) -> str:
         """Format for Sentry reports."""
         return str(self)
 
+    def all_jira_filters(self) -> Iterable[JIRAFilter]:
```

Comment:
```suggestion
    def all_jira_filters(self) -> Iterator[JIRAFilter]:
```

Ufff, I slipped on this so many times. You cannot iterate this more than once.